packages/app/src/store/specs-store.ts

```diff
@@ -4,15 +4,16 @@ import { defineStore } from 'pinia'
 export interface SpecState {
   activeSpec: SpecFile | null | undefined
   specFilter?: string
+  testFilter: Cypress.TestFilter | null
```

Comment:
why default to null vs undefined?

Reply:
Reverted to undefined: [5e431a0](https://github.com/cypress-io/cypress/pull/25265/commits/5e431a05532e2116e048ad3468ede9aa41ff6eda)